model/build/db.go

```diff
@@ -244,6 +244,6 @@ func UpdateAllBuilds(query interface{}, update interface{}) (*mgo.ChangeInfo, er
 func Remove(id string) error {
 	return db.Remove(
 		Collection,
-		bson.M{IdKey: id},
+		ById(id),
```

Comment:
this will remove a random document from the collection, the previous implementation was correct, because of the "db.Q" is not bson serialize.